catalog/systemd.ru.catalog

```diff
@@ -274,7 +274,7 @@ Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
 столбце файла /etc/fstab, либо в параметре Where= файла конфигурации юнита),
 не является пустым. Это никак не мешает монтированию, однако ранее находившиеся
 в нем файлы будут недоступны. Чтобы получить к ним доступ, вы можете вручную
-перемонтировать нижележащую файловую систему в другую точку.
+перемонтировать эту файловую систему в другую точку.
```

Comment:
Using "this filesystem" instead of "underlying filesystem" is probably ambiguous; it is not clear whether "this" refers to filesystem being mounted or filesystem where mount point is located. But using "filesystem where directory is located" is too long :) 

The problem here is that direct translation for "underlying" sounds like artificial Russian word here.